src/Symfony/Component/Workflow/DependencyInjection/ValidateWorkflowsPass.php

```diff
@@ -1,64 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Workflow\DependencyInjection;
-
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Exception\RuntimeException;
-use Symfony\Component\Workflow\Validator\StateMachineValidator;
-use Symfony\Component\Workflow\Validator\WorkflowValidator;
-
-/**
- * @author Tobias Nyholm <tobias.nyholm@gmail.com>
- */
-class ValidateWorkflowsPass implements CompilerPassInterface
```

Comment:
Did we never register the pass inside the FrameworkBundle? Also, removing the class here is a BC break for anyone using it in a standalone setup.

Reply:
Indeed, I forgot to remove it from the framework bundle.
I'm not sure if we should consider it as a BC break for the reason I explained in https://github.com/symfony/symfony/pull/29146#issuecomment-444194286